app.py

```diff
@@ -177,20 +177,65 @@ def api_call_for_news(data):
     socketio.emit('newsData', {
         'newsObjectLst': newsObjectLst
     })
+    
+    
+    
 
     return response_json["articles"]
 
+def trending_news():
+    "Trending news Api call"
+    query = ['trump', 'biden', 'election', 'obama', 'Republicans', 'democrats', 'governer']
```

Comment:
Few comments on the query. 

- Names should be proper, such as "Trump" instead of "trump". Not sure if this makes a huge difference but might result in more searches. 
- Governor is spelled wrong
- Could probably remove the 's' from republicans and democrats, might result in more hits. 

Could add some more words such as:
politics
government
law
state
union
bills
congress